admin/autoupdater/src/autoupdater.c

```diff
@@ -286,7 +294,8 @@ static bool autoupdate(const char *mirror, struct settings *s, int lock_fd) {
 	sprintf(manifest_url, "%s/%s.manifest", mirror, s->branch);
 
 
-	printf("Retrieving manifest from %s ...\n", manifest_url);
+	if (!s->check)
+		printf("Retrieving manifest from %s ...\n", manifest_url);
```

Comment:
I think eventually we should move all human-readable output from stdout to stderr, but for now this is okay.